osu.Game/Tests/Visual/OsuGameTestScene.cs

```diff
@@ -116,9 +116,9 @@ public class TestOsuGame : OsuGame
 
             public new BackButton BackButton => base.BackButton;
 
-            public new BeatmapManager BeatmapManager => base.BeatmapManager;
+            public BeatmapManager BeatmapManager => (BeatmapManager)base.ManagerStorage["BeatmapManager"];
```

Comment:
There is no world wherein I could objectively agree to this sort of change. This is just vastly worse than the previous code due to several reasons (losing strong typing, relying on magic string keys for looking up critical components).